src/templates/new_account/welcome_page.html.tt

```diff
@@ -0,0 +1,48 @@
+<div class="center-text static_full">
+    <span class="account-created"></span>
+
+    [% INCLUDE global/separator_line.html.tt class="gr-padding-5" invisible=1 %]
+
+    <h1>[% l('Congratulations!') %]</h1>
+    <p>[% l('You have successfully created your Virtual Account.') %]</p>
+
+    [% INCLUDE global/separator_line.html.tt class="gr-padding-10" invisible=1 %]
+
+    <div class="box bordered gr-8 gr-12-p gr-centered">
+        <div class="container gr-row box-grid">
+            <div class="gr-7 gr-12-p gr-12-m gr-padding-30 gr-parent">
+                <div class="box border-gray gr-gutter gr-padding-10">
+                    <div>
+                        <strong>[% l('Real Account') %]</strong>
+                        <p class="payment-icons">
+                            <img src="[% request.url_for('images/pages/set_currency/usd_icon.svg') %]">
+                            <img src="[% request.url_for('images/pages/set_currency/aud_icon.svg') %]">
+                            <img src="[% request.url_for('images/pages/set_currency/eur_icon.svg') %]">
+                            <img src="[% request.url_for('images/pages/set_currency/gbp_icon.svg') %]">
+                            <img src="[% request.url_for('images/pages/set_currency/btc.svg') %]">
+                            <img src="[% request.url_for('images/pages/set_currency/ltc.svg') %]">
+                            <img src="[% request.url_for('images/pages/set_currency/bch.svg') %]"></p>
+                        <p class="font-s">[% l('Upgrade to a real account and start trading using a wide range of crypto and fiat currencies.') %]</p>
+                    </div>
+                    <div class="box-item-end">
+                        <p><a class="button" href="[% request.url_for('realws.html') %]"><span>[% l('Upgrade now') %]</span></a></p>
```

Comment:
The URL differs according to the account type which client can upgrade to.
Need to set the URL via js, like other places.
Thanks.

Reply:
Ok, noted. So is it alright if I create a new module for `WelcomePage` in `static_pages.js` to assign the `href` to the buttons in js?

Reply:
Sure, as long as it's not getting so long.